### ISSUE TITLE
IEMOCAP training

### ISSUE THREAD

**Issue by curlneuron** [speechbrain/speechbrain#1296]
When I  use the emotion recognition recipe IEMOCAP, I find that even the training example is setting to 2000 (almost with a  50% cut off ). The accuracy is not change too much(within 1%).
Is it because the overfitting in this corpus? 
Or the pretrained model too good to perform like that.

**Comment by BenoitWang** [comment]
Hi @curlneuron, do you still have the problem? I don't think it's because of the overfitting because the test accuracy is not too bad compared with that of training. Did you use the wav2vec recipe ? And how did you cut the data ?

**Comment by Adel-Moumen** [comment]
Hello,

Is the issue still up, please? Thanks. 🙂
